manual/source/Tutorial/Algorithms/Alignment/PairwiseSequenceAlignment.rst

```diff
@@ -110,17 +75,17 @@ Assignment 1
 
    Objective
 
-     Compute a global alignment between the DNA sequences ``"AAATGACGGATTG"``.
-     ``"AGTCGGATCTACTG"`` using the Gotoh algorithm with the following scoring parameters: ``match = 4``, ``mismatch = -2``, ``gapOpen = -4`` and ``gapExtend = -2``.
-     Store the alignment in an Align object and and print it together with the score.
+     Compute two global alignments between the DNA sequences ``"AAATGACGGATTG"``.
+     ``"AGTCGGATCTACTG"`` using the Gotoh algorithm (Affine Gap model) with the following scoring parameters: ``match = 4``, ``mismatch = -2``, ``gapOpen = -4`` and ``gapExtend = -2``.
+     Store the alignments in two Align objects and print them together with the scores.
 
    Hints
      .. container:: foldable
 
-        The Gotoh algorithm uses an affine gap function.
-        In SeqAn you can switch between linear and affine gap functions using the scoring scheme by setting different parameters for ``gapOpen`` and ``gapExtend``.
+        The Gotoh algorithm uses the Affine Gap function. In SeqAn you can switch between Linear, Affine and Dynamic gap functions using the scoring scheme by setting one of the three tags ``LinearGaps()``, ``AffineGaps()`` or ``DynamicGaps()`` and by setting the ``gapOpen`` and ``gapExtend`` parameters.
+        If the two gap value are equal the Linear Gap model will be selected as default while the Affine Gap model is chosen as standard when the gap costs are different. If the Dynamic Gap model :cite:`Urgese2014` is required the relative tag must be supplied.  
```

Comment:
Probably, your english is better than me. But this sentence is not clear to me.

_If the two gap value are equal the Linear Gap model will be selected as default_